docs/source/public_api/linops.rst

```diff
@@ -1,19 +1,7 @@
+**************
 probnum.linops
-==============
-
-Finite-dimensional Linear Operators.
-
-This package implements finite dimensional linear operators. It can be used to
-represent linear maps between finite-dimensional vector spaces without explicitly
-constructing their matrix representation in memory. This is particularly useful for
-sparse and structured matrices and often allows for the definition of a more
-efficient matrix-vector product. Linear operators support common algebraic
-operations, including matrix-vector products, addition, multiplication, and
-transposition.
-
-Several algorithms in the :mod:`probnum.linalg` subpackage are able to operate on
-:class:`~probnum.linops.LinearOperator` instances.
```

Comment:
why is this text removed?

Reply:
aah is it because it was somehow doubled? (you mention something about this in the PR description)

Reply:
The text is removed so there is only a single source of documentation (the docstrings), otherwise there is a doubling between the `.rst` files and the docstrings in the module level `__init__.py` files. Thanks for noting it though, I had not moved all of these paragraphs to the corresponding docstrings! Should be fixed now.